### DISCUSSION TITLE
Topics for Wiki

### DISCUSSION THREAD

**Discussion by SkyKnight43** [hasseily/Deathlord-Relorded#3] [General]
I am pleased to see the Wiki information here!

Further topics that I think would be interesting are how stats are used in combat. E.g. in hit rolls, is STR or DEX a factor, etc.

I would also like to see an investigation of chest trap avoidance, and of spell effectiveness (How are saves determined, is INT a factor, etc.)

I realize that is a lot of work. I am happy to see any information that is available.

**Comment by hasseily** [comment]
Thanks for the requests. I'll look into those when I am done fixing the auto-rolling. It turns out the FAQs are wrong regarding MIN and MAX of race attributes.

**Reply by SkyKnight43** [reply]
I'm interested in hearing about that! I did notice that the Shukenja hit die is listed as 5, when actually it is 6.

**Reply by hasseily** [reply]
For starting min and max race attributes:
https://github.com/hasseily/Deathlord-Relorded/wiki/Player-Characters#minmax

**Reply by SkyKnight43** [reply]
Very interesting! Thanks for the update.
